node_modules/gh-users/lib/api.js

```diff
@@ -50,11 +50,24 @@ var init = module.exports.init = function(callback) {
  * @param  {Object}         callback.err                    An error object, if any
  * @param  {User}           callback.user                   The requested user
  */
-var getUser = module.exports.getUser = function(id, callback) {
-    // TODO: id validation
-    // TODO: hide calendarToken
+var getUser = module.exports.getUser = function(ctx, id, callback) {
+    var validator = new Validator();
+    validator.check(null, {'code': 401, 'msg': 'You need to be authenticated in order to get a user'}).isLoggedInUser(ctx);
+    validator.check(id, {'code': 400, 'msg': 'A valid user id must be provided'}).isInt();
+    if (validator.hasErrors()) {
+        return callback(validator.getFirstError());
+    }
+
+    UsersDAO.getUser(id, function(err, user) {
```

Comment:
Wouldn't this allow you to get users from a different app? We probably also need a test for that.